component/service/src/main/java/org/exoplatform/social/rest/impl/space/SpaceRestResourcesV1.java

```diff
@@ -1349,10 +1350,44 @@ public Response getSpaceExternalInvitations(@Context UriInfo uriInfo,
     if (space == null ||  (! spaceService.isManager(space, authenticatedUser) && ! spaceService.isSuperManager(authenticatedUser))) {
       throw new WebApplicationException(Response.Status.UNAUTHORIZED);
     }
-    List<SpaceExternalInvitationEntity> spaceExternalInvitations = spaceService.findSpaceExternalInvitationsBySpaceId(id);
+    List<SpaceExternalInvitation> spaceExternalInvitations = spaceService.findSpaceExternalInvitationsBySpaceId(id);
     return EntityBuilder.getResponseBuilder(spaceExternalInvitations, uriInfo, RestUtils.getJsonMediaType(), Response.Status.OK).build();
   }
 
+
+  /**
+   * {@inheritDoc}
+   */
+  @DELETE
+  @Path("{id}/externalInvitations/{invitationId}")
```

Comment:
you don't need to pass spaceId
```suggestion
  @Path("externalInvitations/{invitationId}")
```